core/src/main/java/org/apache/sdap/mudrod/metadata/structure/MetadataExtractor.java

```diff
@@ -67,31 +67,15 @@ public MetadataExtractor() {
    * @param type  metadata type name
    * @return metadata list
    */
-  protected List<PODAACMetadata> loadMetadataFromES(ESDriver es, String index, String type) {
+  protected List<Metadata> loadMetadataFromES(ESDriver es, String index, String type) {
 
-    List<PODAACMetadata> metadatas = new ArrayList<PODAACMetadata>();
+    List<Metadata> metadatas = new ArrayList<Metadata>();
```

Comment:
There is no need to add type safety to the right hand side of the assignment/ Please just use the diamond operator ```<>``` e.g. remove ```Metadata```